### PR TITLE
V1.1.2

### PR DESCRIPTION
**Pull request by AdrielA-GoLogic**

### CODE REVIEW

File: lib/types/sms.d.ts [hunk 1]
```diff
@@ -32,14 +32,15 @@ export interface SmsRecipient {
...
+    statusMessage: string;
 }
 export interface SubmitSmsRequest {
     body: string;
     recipients: Recipient[];
     from: string;
     scheduledDate: Date | null;
-    optOutMessage: boolean;
-    addUnsubscribeLink: boolean;
+    optOutMessage: boolean | false;
```
Comment (csjuddsosa) [inline lib/types/sms.d.ts:42]: No need to type 'addUnsubscribeLink' and 'optOutMessage' to 'false'. 'boolean' is enough

It this 2 are optional, you can do

`
{
...,
  addUnsubscribeLink?: boolean;
  optOutMessage?: boolean;
}
`



then have it default to false on submitFax method

`async submitFax(
    request: SubmitFaxRequest
  ): Promise<BaseResponse<SubmitFaxResponse>> {
    const files = await Promise.all(
      request.documents.map((document) => this.uploadDocument(document))
    );

    if (files.find((file) => !file.success)) {
      throw new NotifyreError(
        'The document upload process failed',
        400,
        files.map((file) => (file.success ? null : file.message))
      );
    }

    const documents: BaseResponse<GetDocumentStatusResponse>[] =
      await this.pollDocumentsStatus(files.map((file) => file.payload!));

    return this.httpClient.post(`${this.basePath}/send`, {
      templateName: request.templateName,
      faxes: {
        clientReference: request.clientReference,
        files: documents.map((document) => document.payload!.id),
        header: request.header,
        isHighQuality: request.isHighQuality,
        recipients: request.recipients,
        scheduledDate: request.scheduledDate
          ? dateToTimestamp(request.scheduledDate)
          : null,
        sendFrom: request.sendFrom,
        senderID: request.sendFrom,
        subject: request.subject,
        addUnsubscribeLink: request?.addUnsubscribeLink || false,
        optOutMessage: request?.optOutMessage || false
      }
    });
  }`
  
  Also submitFax function request doesn't seem inlucde these 2?

Comment (AdrielA-GoLogic) [reply lib/types/sms.d.ts]: i've updated the code based on the review. The optOutMessage and addUnsubscribeLink only applies to submitSMS

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "notifyre-nodejs-sdk",
-  "version": "1.1.1",
+  "version": "1.1.2",
```
Comment (juddfsosagologic) [inline package.json:3]: Version should probably be 1.2.0 since these changes consists of new features and updates and not bugfixes

Comment (AdrielA-GoLogic) [reply package.json]: ok will merge this to the new version branch and create a pr

**Pull request closed without merging**